src/components/SuggestionsProvider/SuggestionsList.js

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import { FlatList } from 'react-native';
+import PropTypes from 'prop-types';
+import styled from '@stream-io/styled-components';
+import { CommandsItem, MentionsItem } from '../AutoCompleteInput';
+
+const Wrapper = styled.TouchableOpacity`
+  height: ${({ height }) => height};
+  position: absolute;
+  width: 100%;
+  z-index: 90;
```

Comment:
alright well let's make sure we have something here that works on android and iOS as we know z-index has issues